lib/boxr/metadata.rb

```diff
@@ -84,5 +84,26 @@ def metadata_schema(scope, template_key)
       schema
     end
 
+    def create_metadata_template(scope, templateKey, displayName, fields: [], hidden: nil)
```

Comment:
Since there's a series of methods specific to Metadata Templates, I think it would be better to split those out from this file so we don't put too many different things in here.

Reply:
Also, parameters and variables in Ruby should be lowercase snake_case